src/styles/LocationDetails.css

```diff
@@ -0,0 +1,24 @@
+.location-details {
+  display: inline-block;
+}
+
+.location-details__heading {
+  position: relative;
+  z-index: 1;
+  font-size: 2.5rem;
+  color: #153253;
```

Comment:
The z-index and color properties can be removed here, the `z-index` on the after element is enough to get the right effect and the color property is just adding the same one thats already being inherited from the body class